pangolin/app/run_alg.py

```diff
@@ -1,4 +1,13 @@
 import os
+import subprocess
+from .logger import get_logger
+
 
 def run_alg(input: str, output: str):
-    os.system(f"pangolin {input} GRCh38.primary_assembly.genome.fa.gz gencode.v38.annotation.db {output}")
\ No newline at end of file
+    process = subprocess.Popen(f"pangolin {input} GRCh38.primary_assembly.genome.fa.gz gencode.v38.annotation.db {output}", shell=True, stdout=subprocess.PIPE)
```

Comment:
- good job with subprocess
- consider using variables from config or env variable in docker-compose for reference files (i know i started with hardcoded file names)